docs/quick_start_python.rst

```diff
@@ -2,7 +2,34 @@ Quick Start Python
 ==================
 
 The use of the Python interface is pretty straight-forward since
-there is only one method: :code:`evaluate(..)`.
+there is only one method: :code:`evaluate(..)` or the alternative
+class :code:`GravityEvaluable` caching the polyhedron data.
+
+The method calls follow the same pattern as the C++ interface. Thus it is always:
+
+.. code-block:: python
+
+    import polyhedral_gravity as model
+
+    #############################################
+    # Free function call
+    #############################################
+    # Define the polyhedral_source which is either (vertices, faces) or a list of files
+    # Define the density (a float)
+    # Define the computation_points which is either a single point or a list of points
+    # Define if the computation is parallel or not (the default is parallel, which corresponds to True)
+    results = model.evaluate(polyhedral_source, density, computation_point, parallel)
+
+    #############################################
+    # With the evaluable class
+    #############################################
+    # Parameters are the same as for the free function call
+    evaluable = model.GravityEvaluable(polyhedral_source, density)
```

Comment:
If possible I would suggest the use of [named argument, helps readability](https://trstringer.com/python-named-arguments/#self-documenting-code)